snc.sh

```diff
@@ -262,6 +283,12 @@ while retry ${OC} get mcp master -ojsonpath='{.status.conditions[?(@.type!="Upda
     echo "Machine config still in updating/degrading state"
 done
 
+# Create a container from baremetal-runtimecfg image which consumed by nodeip-configuration service so it is
+# not deleted by `crictl rmi --prune` command
+${SSH} core@api.${SNC_PRODUCT_NAME}.${BASE_DOMAIN} -- "sudo podman create --name baremetal_runtimecfg ${BAREMETAL_RUNTIMECFG}"
+
```

Comment:
Can we re-pull the image after pruning? This container creation/removal feels hackish.